### PR TITLE
Remove landing pages for attribute categories

### PR DESCRIPTION
**Pull request by queengooborg**
This PR removes the landing pages for the specific categories of attributes.  Cherry-picked from #32366.

### CODE REVIEW

File: files/en-us/web/svg/attribute/index.md [hunk 1]
```diff
@@ -343,26 +343,33 @@ Below is a list of all of the attributes available in SVG along with links to re
...
+- {{SVGAttr("xlink:role")}}
+- {{SVGAttr("xlink:arcrole")}}
+- {{SVGAttr("xlink:title")}}
+- {{SVGAttr("xlink:show")}}
+- {{SVGAttr("xlink:actuate")}}
 
 ### Presentation attributes
 
-> **Note:** All [SVG presentation attributes](/en-US/docs/Web/SVG/Attribute/Presentation) can be used as CSS properties.
+> **Note:** All SVG presentation attributes can be used as CSS properties.
```
Comment (estelle) [inline files/en-us/web/svg/attribute/index.md:372]: `d` is missing from this list

Comment (queengooborg) [reply files/en-us/web/svg/attribute/index.md:372]: Since that's an issue that does not come from this PR, let's fix that in a separate PR!

Comment (estelle) [reply files/en-us/web/svg/attribute/index.md:372]: i added it in because we'll never remember to do it ;).

**Review by estelle — APPROVED** [review]
🎉

**Pull request merged**